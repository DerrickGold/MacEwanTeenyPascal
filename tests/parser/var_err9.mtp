(* variable section start *)

var test: integer;

begin

end
(*missing .*)
