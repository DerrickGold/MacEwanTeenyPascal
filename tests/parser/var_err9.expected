Syntax Error: line 9:
Unexpected token (TOK_ENDFILE, ''), expected a token type of:  
  'TOK_PERIOD'
